support-frontend/assets/helpers/images/imageCatalogue.json

```
@@ -40,6 +40,9 @@
 	"supporterPlusLandingUnitedStates": "527b59919012a7666ccd067a3c82ad63682d2418/0_0_420_165",
 	"weeklyCampaignBenefitsImg": "a9fb03da0efd8e4bc19c48cdcbacc659d887ae4a/0_0_900_600",
 	"weeklyCampaignHeroImg": "c7d642f416d735403c2aab9083eafe4cbd4bd3da/0_0_500_333",
+	"newspapperLandingHeroDesktop": "8063ab98aa49fdecd63b4a4f5b2f9f74f6f1a285/0_0_2092_1568",
```

Sorry, annoying spelling nitpick! (And the two lines below)

```suggestion
	"newspaperLandingHeroDesktop": "8063ab98aa49fdecd63b4a4f5b2f9f74f6f1a285/0_0_2092_1568",
```